API/nearest_neighbors_model.py

```diff
@@ -3,9 +3,9 @@
 from sklearn.neighbors import NearestNeighbors
 from sklearn.feature_extraction.text import TfidfVectorizer
 
-model = pickle.load(open("../models/nearest_neighbors_model.sav", "rb"))
-transformer = pickle.load(open("../models/transformer.sav", "rb"))
-strains = pd.read_csv("../src/data/nn_model_strains.csv")
+model = pickle.load(open("/home/andronik/repos/Data-Science/models/nearest_neighbors_model.sav", "rb"))
```

Comment:
Make sure the path is reproducible for others. It was probably good like it was.